src/frontend/komponenter/Fagsak/Vedtak/RefusjonEøs/RefusjonEøsPeriode.tsx

```diff
@@ -72,17 +68,23 @@ const RefusjonEøsPeriode: React.FC<IRefusjonEøsPeriode> = ({ refusjonEøs, beh
             onOpenChange={håndterLukkingOgÅpningAvPanel}
             content={
                 <FlexColumnDiv>
-                    <RefusjonEøsSkjema skjema={skjema} />
+                    {erRadEkspandert && <RefusjonEøsSkjema skjema={skjema} />}
```

Comment:
Må ha denne for å trigge ny render av skjemaet ved åpning lukking (og dette er for å sikre at datovelgeren rendrer riktig verdi). Litt hacky løsning 🙃